src/app/related-search/related-search.component.spec.ts

```diff
@@ -75,7 +75,18 @@ describe('RelatedSearchComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it('should create an instance', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should display results if any, along with right keyword', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    if (component.results.length > 0) {
+      expect(compiled.querySelector('div.card'));
+      expect(component.keyword).not.toBeNull('Keyword has not been initialized');
+    }
```

Comment:
@Marauderer97 Just asking, did you face the problem `Expect null to be truthy` while testing?